app/views/catalog/_alternate_catalog.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class='alternate-catalog' data-alternate-catalog="<%= articles_path(q: params.fetch(:q)) %>">
+<div class='alternate-catalog' data-alternate-catalog="<%= articles_path(q: params.fetch(:q), f: { eds_search_limiters_facet: ['Direct access to full text'] }) %>">
```

Comment:
I wonder if we could use the `FacetOptionsPresenter` to find the limiter that is `enabled_by_default?` and use that here instead.  We might be able to pull the field reference from that as well.

Reply:
Thanks for the recommendation. Could you talk about that API a bit more? Looking at `FacetOptionsPresenter` I'm not really sure how I should use that in this context.

Reply:
Check out where we set the checkbox on the [article home page](https://github.com/sul-dlss/SearchWorks/blob/master/app/views/articles/_home_page.html.erb#L20-L22) for context.

There is a `facet_options_presenter` that you can use to easily access the facet options object.  So something along the lines of

```ruby
facet_otpions_presenter.limiters.find(&:enabled_by_default?)
```

should get you the the `Limiter` object that has the EDS administration config label (e.g. 'Direct access to full text').  

This will have the benefit of not requiring a change on our end of that text is updated or the default changes.

Reply:
Upon further consideration I decided to remove using the `FacetOptionsPresenter` here. A few reasons:

 - I thought that now needing to instantiate an `Eds:Session` in the `CatalogController` to do a query against the `ArticleController` felt a bit unclean. The ajax query should handle the session stuff, and I don't like the idea of needing to do that just to create a link that ajax will hit.
 - I recognize that the default search behavior is hardcoded. I'm not sure of the right abstraction yet and it didn't feel right from `FacetOptionsPresenter` so I'm leaving hard coded.

Reply:
One of our main goals is to not have to change SearchWorks code if this text changes.

I think we would have to ask Jennifer and Alexis to weigh in because this could potentially have large impacts into approaches that we've already decided on.  I would really like to avoid hard-coding this in our codebase at all costs.

Is there a performance concern you're worried about here?  We can move the EDS session up if needed and the session is cached so it wouldn't be an issue w/ duplicating the session. If it's a naming issue, then let's rename it to make it "feel better".

`FacetOptionsPresenter` is something that does exactly what we want here AFAICT, as well as meets a need of honoring EDS admin settings.  If Jennifer and Alexis are willing to completely break that, then 👍.